base_business_document_import/models/business_document_import.py

```diff
@@ -189,6 +194,23 @@ def _hook_match_partner(
             self, partner_dict, chatter_msg, domain, partner_type_label):
         return False
 
+    @api.model
+    def _match_partner_by_gln(self, gln_number_from_file):
+        """ You may override if your gln_number is not defined
+            by partner_identification_gln module
+        """
+        categ = self.env.ref(
+            "partner_identification_gln."
+            "partner_identification_gln_number_category", raise_if_not_found=False)
+        if categ:
+            # partner_identification_gln is installed
+            id_number = self.env["res.partner.id_number"].search([
+                ("category_id", "=", categ.id),
+                ("name", "=", gln_number_from_file)], limit=1)
+            # GLN unicity is managed by gln module
+            if id_number and id_number.name == gln_number_from_file:
```

Comment:
I think here `id_number.name == gln_number_from_file` is not needed anymore:

```suggestion
            if id_number:
```

Reply:
Oops you're right